bears/js/JSONFormatBear.py

```diff
@@ -10,10 +10,7 @@
 
 
 class JSONFormatBear(LocalBear):
-    try:
-        DecodeError = json.decoder.JSONDecodeError
-    except AttributeError:
-        DecodeError = ValueError
+    DecodeError = json.decoder.JSONDecodeError
```

Comment:
It is already being tested. The coverage information is collated from all the python versions we test against - so, ther python 3.4 test will cover that branch